plugins/logs/plugin.go

```diff
@@ -385,6 +415,37 @@ func (p *Plugin) bufferChunk(buffer *logBuffer, bs []byte) {
 	}
 }
 
+func (p *Plugin) maskEvent(ctx context.Context, event *EventV1) error {
```

Comment:
Is it possible to check first if the policy has been defined? Could maybe avoid some of the extra work being done here if there isn't one set by the deployer.

Reply:
I think if this becomes an issue we could add a setting to disable masking. My intuition is that most users will need masking if they have decision logging enabled.